src/providers/twitch/api/Helix.cpp

```diff
@@ -646,7 +649,61 @@ void Helix::updateChannel(QString broadcasterId, QString gameId,
             successCallback(result);
         })
         .onError([failureCallback](NetworkResult result) {
-            failureCallback();
+            if (!result.status())
+            {
+                failureCallback(Error::Unknown, result.formatError());
+                return;
+            }
+
+            auto obj = result.parseJson();
+            auto message = obj.value("message").toString();
+
+            switch(*result.status())
+            {
+                case 401: {
+                    if (message.startsWith("Missing scope",
+                                           Qt::CaseInsensitive))
+                    {
+                        failureCallback(Error::UserMissingScope, message);
+                    }
+                    else if (message.compare(
+                                 "The ID in broadcaster_id must match the user "
+                                 "ID found in the request's OAuth token.",
+                                 Qt::CaseInsensitive) == 0)
+                    {
+                        failureCallback(Error::UserNotAuthorized, message);
+                    }
+                    else
+                    {
+                        failureCallback(Error::Forwarded, message);
+                    }
+                }
+                break;
+
+                case 400:
+                case 403: {
+                    failureCallback(Error::Forwarded, message);
+                }
+                break;
+
+                case 409:
```

Comment:
This seems wrong. Shouldn't 409 forward the message?

Reply:
According to the [Modify Channel Information](https://dev.twitch.tv/docs/api/reference/#modify-channel-information) docs, 409 for this endpoint will give a Too Many Requests response when trying to see the Branded Content flag too frequently, which wouldn't be hit by `/settitle` or `/settgame` of course but I just tried to handle all errors currently provided in the docs.

Reply:
Oh, right. I read Too Many Requests and thought it would be 429. Probably should say 409 Conflict in the docs.

Reply:
Since it looks like it's just a typo in the docs, I'll remove the 409, and leave it to just 429.